src/task 22/task22.js

```diff
@@ -0,0 +1,30 @@
+function buildTable(arr){
+    var Table = document.querySelector("#table");
+    var headers = Object.keys(arr[0]);
+    var table =document.createElement("table");
+    var headerRow = document.createElement("tr");
+    headers.forEach(function (headerText){
+        var header = document.createElement("th");
+        header.textContent = headerText;
+        headerRow.appendChild(header);
+    });
+    table.appendChild(headerRow);
+    arr.forEach(function(X){
+        var row = document.createElement("tr")
+        Object.values(headers).forEach(function(headerText){
+            var cell = document.createElement("td");
+            if(typeof X[headerText] == "number"){
+                cell.style.textAlign = "right"
+            }else if(X[headerText] == " "){
```

Comment:
It could be not blank, but property not passed in data set. Look above. Please handle the case.